### PR TITLE
Update campaigns

### PR DESCRIPTION
**Pull request by domdomegg**
Add Wikipedia Pages Wanting Photos, as per #17

### CODE REVIEW

**Comment by domdomegg** [comment]
@sivaraam could you review please - leaving a comment is fine if you don't have direct access.

File: campaigns.json [hunk 1]
```diff
@@ -33,11 +33,11 @@
 			"link": "https://commons.wikimedia.org/wiki/Commons:Wiki_Loves_Earth_2020"
 		},
 		{
-			"title": "Wiki Loves Africa",
-			"description": "2020 Theme: \"Africa on the Move or Transport\"",
-			"startDate": "2020-02-15",
-			"endDate": "2020-03-31",
-			"link": "https://commons.wikimedia.org/wiki/Commons:Wiki_Loves_Africa_2020"
+			"title": "Wikipedia Pages Wanting Photos",
+			"description": "Wikipedians add photos to Wikipedia articles lacking images",
```
Comment (sivaraam) [inline campaigns.json:37]: May be rephrase this to "Global campaign to add photos to Wikipedia articles that lack images."

**Comment by domdomegg** [comment]
> Other the description suggestion, this looks good to me. 

Updated

> Side note: I wonder if we should have a separate annual campaigns type for campaigns that happen on particular intervals of time. That way, we don't have to remove/add the campaigns each year.

That's a good point. I've actually undone the change to remove the other campaign so we'll have it there next year, and can update it when we have the new dates/theme etc.

**Comment by sivaraam** [comment]
> We can discuss about the annual campaigns type in a separate issue.

I've opened #19 for this.
